addons/maintenance/tests/test_calendar_with_recurrence.py

```diff
@@ -23,26 +24,30 @@ def test_dblclick_event_from_calendar(self):
             'equipment_id': equipment.id,  # necessary for the tour to work with mrp_maintenance installed
             'maintenance_type': 'preventive',
             'recurring_maintenance': True,
+            'repeat_until': datetime.combine(date.today(), datetime.min.time()) + timedelta(days=8, hours=10),
```

Comment:
Make it simple
```py
{
    ...
    'schedule_date': datetime.now(),
    ...
    'repeat_until': datetime.now() + timedelta(days=8),
    ...
}
```

Reply:
`schedule_date` must have a set amount of seconds for the `drag_and_drop` test. They are not overridden by the operation making the test fail.